package.json

```diff
@@ -214,4 +214,4 @@
     "typescript": "^3.9.5",
     "typescript-json-schema": "^0.50.1"
   }
-}
+}
```

Comment:
Please keep the newline at the end of the file as well.

Reply:
Done. Sorry about that, auto save formatting settings on vscode